src/routes/_layout.svelte

```diff
@@ -1,5 +1,6 @@
 <script>
   import Nav from "../components/Nav.svelte";
+  import GlobalCSS from "../components/GlobalCSS.svelte"; //eslint-disable-line
```

Comment:
```suggestion
  import "../components/GlobalCSS.svelte";
```

By the way, if we want to disable some lint rule, we should specify which specific lint rule we are disabling like ` // eslint-disable-line no-use-before-define` and document the reason so that maintainer can know why it is disabled.